cmd/swarmctl/cluster/inspect.go

```diff
@@ -41,6 +41,7 @@ func printClusterSummary(cluster *api.Cluster) {
 		} else {
 			fmt.Fprintf(w, "  Certificate Validity Duration: %s\n", clusterDuration.String())
 		}
+		fmt.Fprintf(w, "  External CA URLs: %s\n", cluster.Spec.CAConfig.ExternalCAURLs)
```

Comment:
It depends on what you you expect "properly" is. There are a bunch of different ways to do it:

https://play.golang.org/p/39z8AKaQe4